aws/cloudformation/update_certs

```diff
@@ -29,7 +29,7 @@ common_name = ARGV[0]
 required_certs = ARGV.dup
 begin
   Acmesmith::Command.start ['current', common_name]
-rescue => _
+rescue => _e
```

Comment:
Is assigning the exception to a variable necessary? Compared to a bare `rescue`.